general/getting-started-in-tech.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: page
+title: Getting Started in Tech
+---
+
+Technology is a vast subject. It can enable you to write a to-do list on your phone, sell goods online or send a space probe to Neptune. Software is built by developers and codebar is here to teach you how to write such software. Whether you want to make a career of it or acquire some exta skills.
+
+Typically software can fall into two (very broad) categories:
+
+- **User interface** - How to present information to people using the technology and enable them to interact with it. For example:
+    - Displaying your vehicle and responding to joystick inputs in a game of Mario Kart
+    - Displaying your balance and allowing you to make a payment in your bank’s online banking site
+    - Displaying current temperature and allowing you to change the target on your Hive or Nest smart thermostat
+- **Data processing** - How to manipulate the data and perform the task that the software implements. For example:
+    - Calculating the position of your vehicle and detecting collisions
+    - Validating that your account has enough funds to send this payment and processing the payment
+    - Starting or stopping your heating or air conditioning depending on the current temperature and the target you set
+
+At codebar, we concentrate on web technologies for several reasons:
+
+- No initial investment is needed: all the tools you need to learn about web technologies available online for free. And the computer you use doesn’t have to be new or a top of the range model, most computers will perform well, including shared computers in libraries.
+- You can see results very fast.
+- They are ubiquitous: a lot of technologies that started on the web are now used in many other places. Even if you end up working in a domain that has nothing to do with the web, you will likely use or interact with web technologies.
+
+Web technologies are designed in such a way that they tend to more clearly separate user interface work from data processing work. As a result, web developer jobs typically fall into three broad categories:
+- Front-end: work that deals with user interfaces
+- Back-end: work that deals with data processing
+- Full stack: work that combines both aspects
```

Comment:
I would _not_ hyphenate any of the three areas. [People](https://english.stackexchange.com/questions/484817/full-stack-vs-full-stack-back-end-vs-back-end-front-end-vs-front-end) seem to hyphenate when they are adjectives (“_the front-end code_”) and not hyphenate when they are nouns (“_I work on the front end_”).